bin/horton-convert

```diff
@@ -36,15 +36,18 @@ def parse_args():
                                                  ' This only works if the input contains sufficient'
                                                  ' data for the output')
     parser.add_argument('-V', '--version', action='version',
-                        version="%%(prog)s (HORTON version %s)" % version.__version__)
-
+                        try:
+                            version="%%(prog)s (HORTON version %s)" % version.__version__
+                        except ImportError:
+                            raise ImportError('No version.py file found'))
```

Comment:
version.py should normally be present. (See my last post on how to add it.)

Reply:
If this is ok I will leave for people who just try it out after installation

Reply:
ok, until our build and development workflow is simplified, we can have this. It is convenient.